lib/wallaroo/core/windows/panes_range_windows.pony

```diff
@@ -61,63 +62,79 @@ class _PanesSlidingWindows[In: Any val, Out: Any val, Acc: State ref] is
     _range = range
     _slide = slide
     _identity_acc = _agg.initial_accumulator()
+    _highest_seen_event_ts = watermark_ts
 
     (let pane_count, _pane_size, _panes_per_slide, _panes_per_window,
       _delay) = _InitializePaneParameters(range, slide, delay)
-
     _panes = Array[(Acc | EmptyPane)](pane_count)
     _panes_start_ts = Array[U64](pane_count)
     _earliest_window_idx = 0
-    var pane_start: U64 = (watermark_ts - (_delay + _range))
+    var pane_start: U64 = watermark_ts - _delay
+
     for i in Range(0, pane_count) do
       _panes.push(EmptyPane)
       _panes_start_ts.push(pane_start)
       pane_start = pane_start + _pane_size
     end
 
+  fun _is_past_end_ts(event_ts: U64, end_ts: U64) : Bool =>
```

Comment:
Can you move this to the end with the other private methods?

Reply:
Done.